server/server.go

```diff
@@ -773,10 +767,12 @@ func (s *server) AddNode(_ context.Context, req *rpcpb.AddNodeRequest) (*rpcpb.A
 		}
 	}
 
+	nodeFlags[PluginDirKey] = req.GetPluginDir()
```

Comment:
Should this only be populated if `GetPluginDir()` returns a non-empty string?

Reply:
ok that is more consistent